insights/parsers/corosync_cmapctl.py

```diff
@@ -57,3 +53,8 @@ def parse_content(self, content):
             key, value = [item.strip() for item in line.split('=')]
             key_without_parenthese = key.split()[0]
             self[key_without_parenthese] = value
+
+
+@parser(Specs.corosync_cmapctl_on_rhel8)
```

Comment:
By using the `@datasource` as above, you can implement it in one single Parser for both RHEL7 and RHEL8